src/components/fields/SchemaField.js

```diff
@@ -12,9 +12,9 @@ import {
   deepEquals,
   getSchemaType,
 } from "../../utils";
+import DefaultTemplate from "../templates/DefaultTemplate";
```

Comment:
I think it's not necessary to import it here if we decided to pass it from the registry

Reply:
I agree